docs/guide/shop_configuration.md

```diff
@@ -0,0 +1,101 @@
+
+## ERP
+
+[ERP integration](erp_integration/erp_integration.md) requires a Web.Connector license or another webservice interface between ERP and the shop.
+
+|Name|Description|
+|--- |--- |
+|Default Country for Template Debitor|Country used as a default.|
+|Login with Customer Number|When enabled, the login process includes a field for providing the customer number.|
+|Use a template debitor number for this shop|When enabled, a template debitor customer number is used if a customer does not have a customer number from the ERP. Template debitor customer numbers can be defined for each country.|
+|Use a template contact number for this shop|When enabled, a template contact number is used if a customer does not have a customer and contact number from the ERP. Template contact numbers can be defined for each country.|
+|Price requests without customer number|When enabled, a price request is sent to the ERP without customer number. A template debitor is used to calculate prices.|
+|Recalculate prices using the ERP after (seconds)|How long information from the ERP is cached to reduce the traffic towards the ERP. Use a "1 hour" syntax.|
+|Variants handling in the ERP|Handling of variant products. Use:</br>SKU_ONLY if the ERP system uses different SKUs per variant. </br>SKU_AND_VARIANT if the ERP system uses a combination of SKU and variant code for variants.|
+|URL of the Web-Connector|The URL points to the Web.Connector installed by the ERP system. Use an HTTPS connection and make sure that the shop can access this IP and port only.|
+|User name (configured per Web-Connector)|User name for communication with the Web.Connector service.|
+|Password (configured per Web-Connector)|Password for communication with the Web.Connector service.|
+|SOAP Web-Service timeout in seconds|Timeout (web service) for communication with the Web.Connector in seconds.|
+|Timeout towards the ERP-System in seconds|Timeout (ERP) for communication with the Web.Connector in seconds.|
+
+## Fallback configuration for price engine
+
+This fallback configuration is used if no shipping costs are set in price and stock management.
+
+|Name|Description|
+|--- |--- |
+|Fallback costs for shipping|Shipping cost used when the free shipping limit is not reached.|
+|Fallback VAT Code for shipping costs|VAT code for shipping.|
+
+## Basket
+
+|Name|Description|
+|--- |--- |
+|Duration of storing anonymous baskets|How long anonymous baskets are stored (in hours).|
+|Update product data after this time|How long  product data is cached in the basket. Use a "1 hour" syntax.|
+
+## Stored basket
+
+|Name|Description|
+|--- |--- |
+|Display stock as a column|When enabled, stock information is displayed in column, otherwise, it is displayed inline (inside product name column).|
```

Comment:
```suggestion
|Display stock as a column|When enabled, stock information is displayed in a separate column, otherwise, it is displayed inline (inside the product name column).|
```